doc/method/introduction.md

```diff
@@ -44,28 +43,31 @@ taxonomic knowledge.
 Although it is primarily a phylogenetics effort, Open Tree requires a
 reference taxonomy for each of these functions.
 
-In 1, the taxonomy is used for converting OTUs (operational
-taxonomic units, or 'tips') to a canonical form.  Supertree
-construction requires that a source tree OTU be matched with an OTU
-from another tree whenever possible.  This is a nontrivial task.  The
-approach taken is to map OTUs to the reference taxonomy, so that OTUs
-are compared by comparing the taxa to which they map.
+In 1, the taxonomy is used for converting OTUs (operational taxonomic
+units, or 'tips') to a canonical form.  Supertree construction
+requires that a source tree OTU be matched with an OTU from another
+tree whenever possible.  This is a nontrivial task because a single
+taxon can be designated as an OTU in different source trees in
+different ways.  The approach taken is to map OTUs to the reference
```

Comment:
can you clarify what you mean by "different ways"? Do you mean that a different taxon might have different OTU labels, or that a tree might place the taxon in different places?